swan-lake/development-tutorials/ballerina-persist/supported-data-stores.md

```diff
@@ -97,13 +97,13 @@ Below are the ways you can provide the data store for each client generation opt
 1. Integrate the client API generation with the package build.
 
    ```shell
-   $ bal persist add --datastore [mysql/mssql/postgresql]
+   $ bal persist add --datastore [mysql/mssql/postgresql/h2]
    ```
 
 2. One-time generation of the client API against the data model.
 
    ```shell
```

Comment:
```suggestion
   ```
```

Reply:
Fixed in [5e81976](https://github.com/ballerina-platform/ballerina-dev-website/pull/9246/commits/5e81976916a09b9206b6f718416408d5686ab873)